store/sqlstore/scheme_supplier.go

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) 2018-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+package sqlstore
+
+import (
+	"context"
+	"database/sql"
+	"net/http"
+
+	"github.com/mattermost/mattermost-server/model"
+	"github.com/mattermost/mattermost-server/store"
+)
+
+func initSqlSupplierSchemes(sqlStore SqlStore) {
+	for _, db := range sqlStore.GetAllConns() {
+		table := db.AddTableWithName(model.Scheme{}, "Schemes").SetKeys(false, "Id")
+		table.ColMap("Id").SetMaxSize(26)
+		table.ColMap("Name").SetMaxSize(64)
+		table.ColMap("Description").SetMaxSize(1024)
+		table.ColMap("Scope").SetMaxSize(32)
```

Comment:
Possible values currently are `channel` and `team` but I left some spare length just in case. Not really any particular reason beyond that.

Reply:
This is all good. Just a note that given the current set of values this column will use 8 bytes regardless of the setting for max size. However changing the type (if required) is expensive, so that's just something to keep in mind when choosing a size. LGTM though.